onedrive_service/tox.ini

```diff
@@ -21,19 +21,23 @@ envlist =
 [testenv]
 basepython = python3.6
 deps =
+    mock
+    onedrivesdk
     pytest>=3.1,<3.2
 usedevelop = True
-commands = {posargs:py.test -l -vvv '{toxinidir}/tests'}
+commands = {posargs:py.test -l -vvv '{toxinidir}/src/tests'}
```

Comment:
Tests are supposed to be outside of `src`.

Reply:
done